app/javascript/packs/member_facing.js

```diff
@@ -4,6 +4,7 @@ import '../shared/show_errors';
 import '../member-facing/registration';
 import '../member-facing/track_shares';
 import 'whatwg-fetch';
+import 'cookieconsent';
```

Comment:
I'd personally create a separate pack (e.g. `packs/cookieconsent.js`) since `member_facing.js` is already a mess! It would also make it easier to dynamically import (with code splitting) if the member is in the EEA (if we do that manual check). Something like:

```js packs/cookieconsent.js 

if (isEEA()) {
  import('cookieconsent').then(initializeCookieConsent);
}

function initializeCookieConsent() {
  window.cookieconsent.initialize({
    ...
  })
}
```

That way, the `cookieconsent` script is only loaded if the member is in the EEA.

Reply:
I don't mind doing this, but it feels it's not worth it to have the broswer make a separate request just for this. I think it makes sense to load all member-facing js from this pack. I agree we should keep different functionality in different files, but I think having one pack makes sense.